src/collection.rs

```diff
@@ -21,25 +21,24 @@ use zvariant::{Dict, ObjectPath, OwnedObjectPath, Value};
 // Should always be created from the SecretService entry point,
 // whether through a new collection or a collection search
 pub struct Collection<'a> {
-    conn: zbus::Connection,
+    conn: zbus::blocking::Connection,
     session: &'a Session,
     pub collection_path: OwnedObjectPath,
-    collection_proxy: CollectionProxy<'a>,
-    service_proxy: &'a ServiceProxy<'a>,
+    collection_proxy: CollectionProxyBlocking<'a>,
+    service_proxy: &'a ServiceProxyBlocking<'a>,
 }
 
 impl<'a> Collection<'a> {
     pub(crate) fn new(
-        conn: zbus::Connection,
+        conn: zbus::blocking::Connection,
         session: &'a Session,
-        service_proxy: &'a ServiceProxy,
+        service_proxy: &'a ServiceProxyBlocking,
         collection_path: OwnedObjectPath,
     ) -> Result<Self> {
-        let collection_proxy = CollectionProxy::new_for_owned(
-            conn.clone(),
-            SS_DBUS_NAME.to_owned(),
-            collection_path.to_string(),
-        )?;
+        let collection_proxy = CollectionProxyBlocking::builder(&conn)
+            .destination(SS_DBUS_NAME.to_owned())?
```

Comment:
Can this be [`from_static_str`](https://docs.rs/zbus_names/2.0.0/zbus_names/enum.BusName.html#method.from_static_str) instead?